Automattic/wp-calypso
# Revert/eslint 4

We found some issues [on upgrading to ESLint v4](https://github.com/Automattic/wp-calypso/issues/18106). Will need more time to investigate, so in the meantime, I'm reverting the upgrade.

### Testing

* Checkout this branch.
* Add an arrow function as a property of a class. For example, in `client/account-recovery/components/account-recovery-error-message/index.jsx`
* Expected result is that you'll be able to commit your changes and `no-undef` errors are reported. Before this change, the commit hook won't let you add them and would report a `no-undef` issue.
